### PR TITLE
Mark the in-progress period as incomplete on click charts

### PR DESCRIPTION
**Pull request by DennisAlund**
## Why

The "Clicks over time" charts always hook downward at the end:

The final point is the current, still-accumulating period: **today** on daily ranges (7d/30d/90d), the current hour on 24h, the current week on 1y, and the current month on `all`. That period only has clicks counted so far, so it almost always sits below a full period and reads as a real drop when it is just "the day isn't over yet."

## What

Keep the in-progress point visible, but render it so it no longer looks like a decline:

- The run of **complete** periods stays a solid line with the gradient fill.
- The **in-progress** point is connected by a **dashed** segment, gets a **hollow** marker, and carries a `today (in progress)` tooltip (`<title>`).

This is done purely at the render layer, in both chart implementations:

- `src/components/big-chart.tsx` — server-rendered chart used by the dashboard and bundle detail.
- `src/client.ts` `renderTimeline` — client-rendered chart on the link detail page.

The data layer (`ClickRepository.getTimeline` / `getSparkline`) is untouched, so today's clicks remain in the series and the existing bucket-count contract and its tests stay green. I chose this over physically dropping today (which would hide today's activity and break those data-layer tests, which `CLAUDE.md` forbids editing).

## i18n

Added `linkDetail.todayPartial` to all three locales (en/id/sv).

## Tests

New `src/__tests__/unit/big-chart.test.ts` covers: the dashed final segment, the in-progress title, the preserved `today` label, the single-point edge case (no dashed connector), and the empty state. Full suite: 958 passing; `tsc --noEmit` clean.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

https://claude.ai/code/session_017TKFX8qZBoKHjaNozTxFik

---
_Generated by [Claude Code](https://claude.ai/code/session_017TKFX8qZBoKHjaNozTxFik)_

### CODE REVIEW

File: src/components/big-chart.tsx [hunk 1]
```diff
@@ -67,10 +67,21 @@ export const BigChart: FC<BigChartProps> = ({ values, range, t, id }) => {
...
+  // and connect the in-progress point with a dashed segment and a hollow marker.
   const baseY = PAD.t + innerH;
-  const area = `${line} L${lastX.toFixed(1)},${baseY} L${PAD.l},${baseY} Z`;
+  const solidPts = pts.slice(0, n - 1);
+  const solidLine = solidPts
+    .map((p, i) => `${i === 0 ? "M" : "L"}${p[0].toFixed(1)},${p[1].toFixed(1)}`)
+    .join(" ");
+  const area = solidPts.length > 0
+    ? `${solidLine} L${solidPts[solidPts.length - 1][0].toFixed(1)},${baseY} L${solidPts[0][0].toFixed(1)},${baseY} Z`
+    : "";
```
[the remark this answers is not in the document]
Comment (DennisAlund) [reply src/components/big-chart.tsx]: Fixed in b447233. The area guard now matches the solid-line guard (`solidPts.length > 1`), so the gradient area only renders with a real segment to fill under. At `n === 2` no area path is emitted. Added a unit test covering the `n === 2` case (no area) and an `n === 3` control (area present).

File: src/client.ts [hunk 2]
```diff
@@ -937,13 +937,22 @@ function renderTimeline(data) {
...
   var baseY = pad.t + innerH;
-  var area = line + 'L' + lastX.toFixed(1) + ',' + baseY + ' L' + pad.l + ',' + baseY + ' Z';
+  var solidLine = '';
+  for (var i = 0; i < n - 1; i++) {
+    solidLine += (i === 0 ? 'M' : 'L') + pts[i][0].toFixed(1) + ',' + pts[i][1].toFixed(1) + ' ';
+  }
+  var area = '';
+  if (n > 1) {
+    area = solidLine + 'L' + pts[n - 2][0].toFixed(1) + ',' + baseY + ' L' + pts[0][0].toFixed(1) + ',' + baseY + ' Z';
+  }
```
[the remark this answers is not in the document]
Comment (DennisAlund) [reply src/client.ts:954]: Fixed in b447233. Changed the area construction to `if (n > 2)` so it matches the solid-line render condition; the degenerate single-point area is no longer emitted. The mirrored `BigChart` fix carries the unit coverage (`client.ts` `renderTimeline` is browser DOM code without a unit harness).
